docs/pages/blog/mui-x-v5.md

```diff
@@ -9,7 +9,11 @@ card: true
 
 We are excited to introduce [MUI X v5.0.0](https://github.com/mui-org/material-ui-x/releases/tag/v5.0.0)!
 
-[MUI X](/x/) is our collection of advanced components (with both MIT and commercially licensed "Pro" features), that includes a data grid, date picker, and tree view. This release continues our focus on making the data grid the best in the industry. Read on to learn more!
+[MUI X](/x/) is our collection of advanced components (with both MIT and commercially licensed "Pro" features).
+Currently, it only includes a data grid.
```

Comment:
@oliviertassinari They are listed on the X page as released. I questioned it in the X page update PR: https://github.com/mui-org/material-ui/pull/28964#pullrequestreview-794371959